README.md

```diff
@@ -50,11 +67,11 @@ To build the project and generate the bundle use Maven
 
     mvn clean install
 
-If everything checks out, the bundle for R should be available in the `target` folder
+If everything checks out, the bundle for `R` should be available in the `target` folder
 
 ## Using R With Nexus Repository Manager 3
 
-After installing the plugin, we have detailed instructions on [how to get started here!](docs/R_USER_DOCUMENTATION.md)
+Please view detailed instructions on [how to get started here!](https://help.sonatype.com/repomanager3/formats/r-repositories)
 
 ## Compatibility with Nexus Repository Manager 3 Versions
 
```

Comment:
Boyscout: If you're feeling generous, I think there should be a : after against in the next line text.

Reply:
d22e5627912bce02daf2b092028c31a797a384e2